lib/classes/config-schema-handler/resolve-ajv-validate.js

```diff
@@ -45,7 +46,18 @@ const getValidate = async (schema) => {
     // See: https://github.com/ajv-validator/ajv/issues/1390#issuecomment-763138202
     ajv.opts.code.formats = Ajv._`require("ajv-formats/dist/formats").fullFormats`;
     ajv.addKeyword(require('./regexp-keyword'));
-    const validate = ajv.compile(schema);
+    let validate;
+    try {
+      validate = ajv.compile(schema);
+    } catch (err) {
+      if (err.message && err.message.includes('strict mode')) {
+        throw new ServerlessError(
+          'Schema does not satisfy "strict" validation mode. Please make sure that all plugins that you are using extend schema in a way that satisfies "strict" mode.',
```

Comment:
Unfortunately, we don't really have a good way of detecting which plugin is causing the issue so the message is still a bit problematic/hard to act on for users - I'm open to potential ideas for improvement here.

Reply:
Is this error happening when a plugin defines a JSON schema that is invalid?

Reply:
Yes, that's correct, it happens for a few plugins and all of them already have a PR prepared (some of them didn't merge yet though)

Reply:
👍  How about something like this:

```suggestion
          'One plugin defines a validation schema that is invalid. Try disabling plugins one by one to identify the problematic plugin and report it to the plugin maintainers.',
```

I'm still not 100% satisfied with this but I have a hard time coming up with something better.

Reply:
That also sounds good, but maybe let's word it as `At least one plugin...` because there might be more than one. What do you think?

Reply:
👍